content/1_docs/1_guide/20_kirby-and-privacy/guide.txt

```diff
@@ -0,0 +1,63 @@
+Title: Kirby & Privacy
+
+----
+
+Description: Things you need to know about Kirby in terms of privacy and GDPR compliance.
+
+
+----
+
+Text:
+
+<info>
+The following information might be incomplete. Your site may collect or process personal data in other ways.
+
+We cannot give any legal advice what this means for you and your sites. If you are unsure what you need to do to comply with GDPR in particular or any other national privacy laws in your country, please consult a lawyer.
+</info>
+
+## Preface
+
+We are not responsible how you handle data privacy on your own Kirby-powered website(s). You are your own data controller and if you are located in the EU or if you have customers in the EU, you must make sure that your site is GDPR-compliant. However, we want to give you the necessary technical information to make it easier for you to comply with GDPR or any other privacy regulations.
+
+## Privacy by design
+
+Kirby and the Panel are designed with privacy in mind, which is why the Kirby core and Panel do not communicate with third-party services or our own servers in any form (exception: licensing, see below). There are no trackers, no analytics and no cookies except those related to sessions and user management. We have no access to your Panel, to your data, to your server or to your users. We also do not use external services to provide web fonts, JavaScript libraries or any other assets for the Panel. Everything runs on your server and stays under your control.
+
+Kirby sites usually do not store or process any personal data of your site visitors. However, here is a list of Kirby features that process data if you use them and some general information about where personal data might get stored.
+
+## User handling
+
+User data gets stored in account files within the `site/accounts` directory. Kirby will create a session cookie to keep track of the currently logged in user if you use a login form in your site’s frontend or if you use the Panel. 
+
+The user credentials are transmitted via HTTP. Therefore, we strongly recommend to use TLS encryption for your sites to protect the passwords and other personal data of your visitors and users.
+
+## Panel
+
+The Panel sets a session cookie for every visitor who uses the Panel to keep track of the currently logged in user. Additionally, the Panel stores unsaved changes in `localStorage`.
+
+## Licensing
+
+To register your license, the Panel connects to our licensing server to verify your license. When you do so, the license server stores the domain for which the license is registered on our license server.
```

Comment:
This doesn't really fit into the list of Kirby features that process data. But to be honest I don't know where it would fit better.

We should also add which information in particular gets transmitted to the server:

- license key
- email address
- domain of the Kirby installation

and when (only once when the license is registered).